pom.xml

```diff
@@ -44,12 +44,14 @@
         <dependency>
             <groupId>com.hazelcast</groupId>
             <artifactId>hazelcast</artifactId>
-            <version>3.11.6</version>
+            <version>3.11.2</version>
         </dependency>
         <dependency>
-            <groupId>com.ikentoo</groupId>
+            <groupId>com.lightspeed-hospitality</groupId>
```

Comment:
They actually didn't change the group id, only the owner of the repo. The pom still says:

```
<groupId>com.ikentoo</groupId>
    <version>0.0.1-SNAPSHOT</version>
    <artifactId>hazelcast-aws-ecs</artifactId>
```

Reply:
oki, will change that.